pkg/controller/common/controller_context.go

```diff
@@ -55,7 +58,16 @@ func CreateControllerContext(cb *clients.Builder, stop <-chan struct{}, targetNa
 	kubeSharedInformer := informers.NewSharedInformerFactory(kubeClient, resyncPeriod()())
 	kubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), targetNamespace, nil)
 	openShiftConfigKubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), "openshift-config", nil)
-	apiExtSharedInformer := apiextinformers.NewSharedInformerFactory(apiExtClient, resyncPeriod()())
+	// filter out CRDs that do not have the MCO label
+	filterObjectChanges := func(opts *metav1.ListOptions) {
+		ls, err := labels.ConvertSelectorToLabelsMap(opts.LabelSelector)
+		if err != nil {
+			glog.Warningf("unable to convert selector %q to map: %v", opts.LabelSelector, err)
+			return
+		}
+		opts.LabelSelector = labels.Merge(ls, map[string]string{"mco-built-in/crd": ""}).String()
```

Comment:
question: i feel like im missing something in this line: we're merging the labelsmap with the mco-build-in label you made? could you explain?

Reply:
In this line we are merging the (new) MCO label with any existing labels, so we don't override them accidentally.

Reply:
after chatting realized, i got confused by the name of the func here, something like assignFilterLabels might be clearer.